### DISCUSSION TITLE
Integration AIO-H3-10.0-E

### DISCUSSION THREAD

**Discussion by revajcagro** [StealthChesnut/HA-FoxESS-Modbus#61] [Q&A] [answered]
[H3_inverter.pdf](https://github.com/StealthChesnut/HA-FoxESS-Modbus/files/10055344/H3_inverter.pdf)

Would anyone be interested in making an integration and extension for a three-phase inverter?

I did get the registry table, but I'm struggling a bit with it.

**Comment by Daibutt** [comment, marked as answer]
Have you seen this fork?
https://github.com/rsaemann/HA-FoxESS-H3-Modbus

**Reply by revajcagro** [reply]
Thank you very much, I didn't know about this fork

**Comment by domerich** [comment]
I got it to work with the work with RS485 over WIFI. My docu is on facebook 
https://www.facebook.com/groups/foxessownersgroup/permalink/1761656834216491/?__tn__=*F
